components/01-atoms/controls/base/control.twig

```diff
@@ -8,6 +12,15 @@
     href: control__url,
   }) %}
   {% set control__element = 'a' %}
+{% else %}
+  {% set control__base_class = control__base_class|default('button') %}
+  {% set control__element = control__element|default('button') %}
+{% endif %}
+
+{% if aria_expanded %}
```

Comment:
`aria_expanded` should only ever apply to a `<button>` correct? I could see this being moved up inside the else but I guess it doesn't need to be.

Reply:
Makes sense. I moved it in and made sure everything still worked as expected. Thanks!